openmoc/numpy_typemaps.i

```diff
@@ -24,6 +24,23 @@
  * using NumPy arrays */
 %apply (double* IN_ARRAY1, int DIM1) {(double* xs, int num_groups)}
 
+/* The typemap used to match the method signature for the Cell rotation
+ * angle setter method. This allows users to set the rotation angles
+ * using NumPy arrays */
+%apply (double* IN_ARRAY1, int DIM1) {(double* rotation, int num_axes)}
```

Comment:
Is there a good reason for using arrays rather than vectors everywhere? If this was a vector, no typemap would be needed.